java-generic-mqtt/src/main/java/com/cumulocity/generic/mqtt/client/GenericMqttPublisher.java

```diff
@@ -0,0 +1,17 @@
+package com.cumulocity.generic.mqtt.client;
+
+import com.cumulocity.generic.mqtt.client.model.GenericMqttMessage;
+
+/**
```

Comment:
suggestion: no need to provide this class' name.
Interface for publishing messages to the server. \<code\>GenericMqttPublisher\</code\> instances are configured and created using {@link GenericMqttFactory#buildPublisher(GenericMqttConnectionConfig)}.

Alternative to 'configured and created' could be simply 'setup'

Reply:
Done.